lib/Field/useLoader.js

```diff
@@ -1,15 +1,30 @@
-import { useState } from 'react';
+import { useEffect, useRef, useState } from 'react';
 
 export function useLoader(action, isInitiallyLoading = false) {
   const [isLoading, setIsLoading] = useState(isInitiallyLoading);
 
+  const isMounted = useRef(true);
+
+  useEffect(
+    () => () => {
+      isMounted.current = false;
+    },
```

Comment:
💚 You could one liner this one with
```suggestion
    () => () => isMounted.current = false,
```
but obvs not blocking as stylistic!

Reply:
I had that originally but my IDE was complaining, I think it's because the non-block arrow function returns expression, whereas it's just expecting a void return type. I think I'll leave it for now 🙂
<img width="1020" alt="Screenshot 2020-06-17 at 10 16 50" src="https://user-images.githubusercontent.com/3472717/84879879-b5788400-b083-11ea-9c3a-1c5c13382069.png">